### DISCUSSION TITLE
Prediction when next release?

### DISCUSSION THREAD

**Discussion by beedell-roke** [NotImplementedLife/FlipnoteDesktop#2] [Q&A] [answered]
It's been in pre-release since 2020.

**Comment by NotImplementedLife** [comment, marked as answer]
Hi there! The old development has been abandoned due to ethical concerns regarding signing flipnotes and sharing them online with possibly forged author ID's. 

However, there's no need to worry. I'm currently testing an alternate method that involves exporting frames as DSi Camera pictures and re-assembling them manually on the DSi using the image import feature. This way, we can preserve the authenticity of the flipnote while still making use of a PC tool in order to create it. This idea has the potential to revive the project. 

Since I'm redoing everything from scratch, I'm trying to include features that feel closer to animation editing rather than just a multiframe paint app.

Here's a screenshot of what it will look like. Work is on going right now after a long pause so you can expect announcement news in the near future!

![](https://github.com/NotImplementedLife/Flipnote.NET/assets/70803115/963553a8-6c5a-46fb-b5a6-a45fceb16065)
